staging/src/k8s.io/apiserver/pkg/registry/generic/rest/streamer.go

```diff
@@ -28,7 +29,7 @@ import (
 )
 
 // LocationStreamer is a resource that streams the contents of a particular
-// location URL
+// location URL. Redirects are forbidden, and treated as a server-side error.
```

Comment:
I get why this is the case for the pods/log endpoint, but it's not immediately obvious why a "LocationStreamer" would forbid redirects. Can we rename this to `LocalLogsStreamer`, or else put `PreventRedirects bool` on the struct? Or if it actually does make sense for something called a "LocationStreamer" to never redirect, could you add a sentence here why that is?

Reply:
I just exposed the RedirectChecker instead, and made the prevention explicit for logs.